docs/install.rst

```diff
@@ -53,13 +50,32 @@ are developing pyFF or want to run multiple python-based applications in
 parallell without having to worry about conflicts between packages.
 
 .. code-block:: bash
-
+  
+  # cd $HOME
   # apt-get install python-virtualenv
-  # mkdir -p /opt/pyff
-  # virtualenv /opt/pyff --no-site-packages
+  # mkdir python-pyff
+  # /opt/rh/rh-python36/root/usr/bin/virtualenv python-pyff --no-site-packages
```

Comment:
Avoid referencing your particular install paths for python. Also https://stackoverflow.com/questions/23842713/using-python-3-in-virtualenv has an article on how virtualenv has changed in python3.